src/supertux/menu/debug_menu.cpp

```diff
@@ -71,6 +71,7 @@ DebugMenu::DebugMenu() :
   add_toggle(-1, _("Use Bitmap Fonts"),
              []{ return g_debug.get_use_bitmap_fonts(); },
              [](bool value){ g_debug.set_use_bitmap_fonts(value); });
+  add_toggle(-1, _("Show Tile Ids In Editor"), &g_debug.show_tile_ids_in_editor);
```

Comment:
In the displayed string, "IDs" would look better IMO.

Also, I propose putting "Show Tile IDs in Editor Toolbox", because it mentions the toolbox specifically.